src/servers/src/http.rs

```diff
@@ -477,6 +479,8 @@ impl HttpServer {
             routing::get(handler::health).post(handler::health),
         );
 
+        router = router.nest("/dashboard", dashboard());
```

Comment:
question: Can we disable the `/dashboard` endpoint by cli options? Maybe in some scenarios, the dashboard service will deploy as the standalone service, and no need to expose the `/dashboard` endpoint.